bundles/org.openhab.binding.deconz/src/main/java/org/openhab/binding/deconz/internal/BindingConstants.java

```diff
@@ -81,23 +105,15 @@
 
     public static final String UNIQUE_ID = "uid";
 
-    public static String url(String host, int port, @Nullable String apikey, @Nullable String endpointType,
-            @Nullable String endpointID) {
+    public static String buildUrl(String host, int port, String... urlParts) {
```

Comment:
Why is this method in the BindingConstants class? Can you move it to a utility class instead?

Reply:
Moved. I  just kept it where it was.